### PR TITLE
[ Milestone 1 ]Add react app starter files

### PR DESCRIPTION
**Pull request by CakeInTech**
- Added Linters to the project.
- Used the `Create-react-app` with the  `npx` command to add starter files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+
+## Set-up linters in your local env
+
+### ESLint
+
+1. Run 
+    ```
+    npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x  @babel/preset-react@7.x @babel/preset-react@7.x
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please edit your readme file by using this Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) with your project and personal data to make your README file more professional.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @MohamedCK,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Gitflow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (muneebulrehman) [inline README.md:37]: - Optional.
Kindly update these sections as well, and the sections which you feel are not relevant to this project, you can remove them to keep the README file clean.

**Review by muneebulrehman — APPROVED** [review]
Hi @MohamedCK , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**